model/src/pyrenew/regression.py

```diff
@@ -10,8 +10,8 @@
 
 import numpyro
 import numpyro.distributions as dist
+import pyrenew.transformation as nt
```

Comment:
As above, should it be just `t` or `pt` now that it's our own module?

Reply:
Fixed here ef26272.